spidermon/contrib/scrapy/extensions.py

```diff
@@ -149,6 +149,14 @@ def _count_item(self, item, skip_none_values, item_count_stat=None):
                 self._count_item(value, skip_none_values, field_item_count_stat)
                 continue
 
+            if isinstance(value, list):
+                items_count_stat = f"{field_item_count_stat}/_items"
+                for list_item in value:
+                    self.crawler.stats.inc_value(items_count_stat)
+                    if isinstance(list_item, dict):
```

Comment:
Should we also count nested lists?

Reply:
If we don't, this could definitely improve performance.